base_tier_validation/models/tier_review.py

```diff
@@ -89,5 +94,22 @@ def _compute_reviewer_ids(self):
         for rec in self:
             rec.reviewer_ids = rec._get_reviewers()
 
+    @api.depends("reviewer_ids", "reviewer_group_id")
+    def _compute_todo_by(self):
+        """ Show by group or by abbrev list of names """
+        num_show = 3  # Max number of users to display
+        for rec in self:
+            if rec.reviewer_group_id:
+                todo_by = _("Group %s") % rec.reviewer_group_id.name
+            else:
+                todo_by = ", ".join(
+                    rec.reviewer_ids[:num_show].mapped("display_name")
+                )
+                num_users = len(rec.reviewer_ids)
+                if num_users > num_show:
+                    todo_by = "{} (and {} more)"\
```

Comment:
This should be translatable (and then you should use %s substitution because of https://github.com/OCA/pylint-odoo/issues/302)

Reply:
Done. Thanks!